packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -10,6 +10,7 @@ import 'binding.dart';
 import 'focus.dart';
 import 'framework.dart';
 import 'overlay.dart';
+import 'pages.dart';
```

Comment:
this is a layering violation. The API is designed so that Navigator doesn't know about Routes.